quality/fca/cypress/support/usager/steps/contact-support-steps.ts

```diff
@@ -3,13 +3,8 @@ import { Then } from '@badeball/cypress-cucumber-preprocessor';
 import { ChainableElement } from '../../common/types';
```

Comment:
question: why there are changes in some of the snapshots? I am not able to see the changes.

Reply:
There has been some slight layout update.